corehq/apps/mobile_auth/views.py

```diff
@@ -64,9 +64,27 @@ def fetch_key_records(request, domain):
         last_issued = string_to_datetime(last_issued).replace(tzinfo=None)
     user_id = request.couch_user.user_id
     payload = FetchKeyRecords(domain, user_id, last_issued).get_payload()
+    device_id = request.GET.get('device_id')
+    if device_id:
+        _touch_user_device_id_last_used(request.couch_user, device_id)
```

Comment:
it should be since the migration is called automatically in the `wrap` function. though thinking about this a bit more I'm inclined to not deal with the migration and make a new property. will update this.